pkger/parser.go

```diff
@@ -180,19 +187,27 @@ func parse(dec decoder, opts ...ValidateOptFn) (*Pkg, error) {
 	return &pkg, nil
 }
 
+// Kind describes the metadata and raw spec for a package kind.
+type Kind struct {
```

Comment:
I would just call it pkger or PkgerItem. Kind is really confusing. And that kindType should be kind

Reply:
point taken. I'm not a fan of the `Item` word per say, nor hte stutter in the name. I'll look at k8s and a few other places to get a feel for their naming convention. I am calling it Kind here b/ that is what k8s calls it, felt better to align than diverge on this. The `KindType`, totally agree, confusing.